doc/doc-support/lib-function-locations.nix

```diff
@@ -44,7 +53,7 @@ let
     builtins.filter
       (elem: elem.value != null)
       (nixpkgsLib.lists.flatten
-        (locatedlibsets nixpkgsLib));
+        (locatedlibsets library));
```

Comment:
Is this generic? doesn't `locatedLibsests` assume that you have the following library structure:

````
# lib
{
  inherit (attrs) concatAttrs;
  attrs = {
    concatAttrs = ...;
  }
}
````
(2 Level nesting, with "categories")

I am unsure if one could also pass

````
# pkgs.writers
{
    writeShell = ...;
}
````

(1 Level nesting, no categories)